lib/status/status.go

```diff
@@ -357,7 +362,31 @@ func probeErrorDetail(p pb.Probe) string {
 	if p.Detail == "" {
 		detail = p.Checker
 	}
-	return fmt.Sprintf("%v failed", detail)
+	// state directory disk space checker always checks /var/lib/gravity
+	// which is default path inside planet but may be different on host
+	// so determine the real state directory if needed
+	if strings.HasPrefix(p.Checker, monitoring.DiskSpaceCheckerID) {
+		var data monitoring.HighWatermarkCheckerData
+		err := json.Unmarshal(p.CheckerData, &data)
+		if err != nil {
+			logrus.Error(trace.DebugReport(err))
```

Comment:
Meh, the function should return error and let the caller decide

Reply:
The error would then propagate all the way up to a bunch of methods that also don't return it. I'd rather just log it here, it's only used in this module.